static/scripts/courses.js

```diff
@@ -369,4 +370,98 @@ $(document).ready(() => {
 
 		$bbbReloadInfoModal.appendTo('body').modal('show');
 	});
+
+	const compareCourseDates = () => {
+		let firstDate = $('#startDate').val();
+		let lastDate = $('#untilDate').val();
+		const lang = getCookie('USER_LANG');
+		const dateFormat = {
+			de: 'DD.MM.YYYY',
+			en: 'MM/DD/YYYY',
+		};
+
+		if (lang === 'en') {
+			firstDate = moment(firstDate, dateFormat.en);
+			lastDate = moment(lastDate, dateFormat.en);
+		} else {
+			firstDate = moment(firstDate, dateFormat.de);
+			lastDate = moment(lastDate, dateFormat.de);
+		}
+
+		if (moment(firstDate).isSameOrBefore(lastDate)) return false;
+
+		return true;
+	};
+
+	const setValidity = (element, errorMessageElement, showError = true) => {
+		if (showError) {
+			element.setCustomValidity('The input is required');
+			if (errorMessageElement) {
+				$(errorMessageElement).css('visibility', 'visible');
+			}
+		} else {
+			element.setCustomValidity('');
+			if (errorMessageElement) {
+				$(errorMessageElement).css('visibility', 'hidden');
+			}
+		}
+	};
+
+	$('#nextSection').on('click', (e) => {
+		e.stopPropagation();
+
+		const selectedOptionsArray = $('#courseTeacher').val();
+		const startDateElement = $('#startDate');
+		const input = $('.chosen-search-input')[0];
+
+		if (compareCourseDates()) {
+			setValidity(startDateElement[0], '#invalidTimeError', true);
+		} else {
+			setValidity(startDateElement[0], '#invalidTimeError', false);
+		}
+
+		if (selectedOptionsArray.length < 1) {
+			setValidity(input, '#courseTeacherErr', true);
+			$('.chosen-search-input').css('box-shadow', 'none');
+			$('#courseTeacher_chosen').addClass('validateError');
```

Comment:
Could you please do the style modifications in the setValidity function. Try to find necessary DOM elements with jquery